scripts/calculateRevenue/protocols/celo/index.ts

```diff
@@ -0,0 +1,29 @@
+import { NetworkId } from '../../../types'
+import { getNearestBlock } from '../utils/events'
+import { fetchTotalGasUsed } from '../utils/networks'
+
+export async function calculateRevenue({
+  address,
+  startTimestamp,
+  endTimestamp,
+}: {
+  address: string
+  startTimestamp: Date
+  endTimestamp: Date
+}): Promise<number> {
+  const startBlock = await getNearestBlock(
+    NetworkId['celo-mainnet'],
+    startTimestamp,
+  )
+  const endBlock = await getNearestBlock(
+    NetworkId['celo-mainnet'],
+    endTimestamp,
+  )
+
+  return await fetchTotalGasUsed({
+    networkId: NetworkId['celo-mainnet'],
+    users: [address],
```

Comment:
We could adjust `fetchTotalGasUsed` to take just one user, but as this function can support multiple users, this retains functionality while conforming to the expected `calculateRevenue` parameters.